algorithms/rainbow/model.py

```diff
@@ -18,8 +20,8 @@ def __init__(self, args, action_space):
         super().__init__()
         self.atoms = args.atoms
         self.action_space = action_space.n
-        self.linear_in = self.get_linear_size(args)
-
+        self.linear_in = self.get_linear_size(args.resolution)
+        # TODO: check what happens if RGB. It might be 3 * history_lenght. Adapt code
```

Comment:
typo lenght to length

Reply:
also im really curious how frames stack on each other. 4 * history length of RGB seems strange to just stack 12 channels on top of each other